libs/sdk-backend-spi/src/common/paging.ts

```diff
@@ -17,4 +17,12 @@ export interface IPagedResource<TItem> {
      * @returns promise of a paged resource with the results of next page
      */
     next(): Promise<IPagedResource<TItem>>;
+
+    /**
+     * Request a specific page of the resource
+     * @param pageIndex - index of requested page: positive, zero-based
+     *
+     * @returns promise of a paged resource with the results of selected page
+     */
+    goTo(pageIndex: number): Promise<IPagedResource<TItem>>;
```

Comment:
Should this throw on negative numbers?

Reply:
I'm not sure, we don't throw on init (withLimit, withOffset) so not sure if we should do here (all throw on underlying layer)
```
    public withLimit(limit: number): IWorkspacesQuery {
        this.limit = limit;
        return this;
    }

    public withOffset(offset: number): IWorkspacesQuery {
        this.offset = offset;
        return this;
    }
```

Reply:
Good point, let's not change it :)